pkg/sentry/pgalloc/pgalloc.go

```diff
@@ -153,7 +153,7 @@ type MemoryFile struct {
 	// operations. This allows MemoryFile.MapInternal to avoid locking in the
 	// common case where chunk mappings already exist.
 	mappingsMu mappingsMutex
-	mappings   atomic.Value
```

Comment:
Ditto, I think we should not change `mappings` as well, since it will additionally cause the slice variable to escape to heap.